### PR TITLE
Create ru-RU.dart

### PR DESCRIPTION
**Pull request by ValeraVvV**
ready

### CODE REVIEW

File: ru-RU.dart [hunk 1]
```diff
@@ -0,0 +1,845 @@
...
+    "btn_bind_via" :  "Привязать через %s" ,
+    "btn_reset_password_via" :  "Сброс через %s" ,
+    "btn_bind_account" :  "Привязать учетную запись" ,
+    "prompt_password_limit_tips" :  "Пожалуйста, введите 6-20 цифр и английских символов, а не чистые числа" ,
+    "prompt_empty_password" :  "Пустой пароль" ,
+    "title_bind_social_account" :  "Привязка социальной учетной записи" ,
+    "prompt_social_account_auth_failed" :  "Авторизация не удалась" ,
+    "prompt_bind_social_account_tips" :  "*Вы можете войти в INMOTION с привязанной социальной учетной записью." ,
+    'state_bound' :  'Связанный' ,
+    «state_not_bound» :  «Не привязано» ,
```
Comment (inmotion-global) [inline ru-RU.dart:108]: Please correct this line.

File: ru-RU.dart [hunk 2]
```diff
@@ -0,0 +1,845 @@
...
+    "btn_common_problem" :  "Часто задаваемые вопросы" ,
+    "page_message_center" :  "Центр сообщений" ,
+    "title_sys_message" :  "Системное сообщение" ,
+    "title_social_message" :  "Социальное сообщение" ,
+    "btn_save_and_share" :  "Сохранить и поделиться" ,
+    "prompt_saved" :  "Сохранено" ,
+    "prompt_save_failed" :  "Не удалось сохранить" ,
+    "prompt_updated" :  "Обновлено" ,
+    "prompt_update_failed" :  "Обновление не выполнено" ,
+    "prompt_data_empty" :  "Данные отсутствуют". ,
```
Comment (inmotion-global) [inline ru-RU.dart:50]: Please remove the '.' at end of this line.

File: ru-RU.dart [hunk 3]
```diff
@@ -0,0 +1,845 @@
...
+    "prompt_wait_for_activation_tips_finished" :  "Пожалуйста, прочтите/посмотрите напоминания о безопасности." ,
+    "prompt_had_read_activation_tips" :  "Я внимательно прочитал приведенные выше напоминания о безопасности и знаю о возможных рисках" ,
+    "btn_activate" :  "Активировать сейчас" ,
+    "prompt_activated" :  "Активировано успешно!" ,
+    "prompt_activate_failed" :  "Не удалось активировать" ,
+    "prompt_activate_canceled" :  "Отменено" ,
+    "prompt_authorize_as_owner_after_activation" :  "Аутентификация колеса требуется для таких операций, как блокировка. Вы хотите подтвердить что вы владельц?" ,
+    "prompt_authorize_as_owner_later_after_activation" :  "Вы также можете сделать это позже в разделе Профиль - авторизация владельца" ,
+    "prompt_set_vehicle_password_failed" :  "Не удалось установить пароль" ,
+    "prompt_set_vehicle_password_later_after_activation" :  "Вы также можете сделать это позже в разделе - Настройки колеса - Пароль"" ,
```
Comment (inmotion-global) [inline ru-RU.dart:139]: Please remove extra double quotes from this line。

File: ru-RU.dart [hunk 4]
```diff
@@ -0,0 +1,845 @@
...
+    "prompt_why_need_location_permission" :  "Зачем INMOTION требуется разрешение на определение местоположения?" ,
+    "prompt_why_do_i_couldnt_find_vehicles" :  "Почему я не могу найти устройство?" ,
+    "btn_view_solutions" :  "Варианты решений" ,
+    "prompt_connect_failed_server_ip_service_error" :  "Временная ошибка сервера, мы скоро это исправим, повторите попытку позже." ,
+    "prompt_connect_failed_unknown_model" :  "Вы не можете использовать приложение INMOTION, потому что модель текущего устройства недействительна. \n Пожалуйста, свяжитесь с нами для получения дополнительной информации." ,
+    "prompt_connect_failed_unknown_sn" :  "Вы не можете использовать приложение INMOTION, потому что серийный номер текущего автомобиля недействителен. \n Пожалуйста, свяжитесь с нами для получения дополнительной информации." ,
+    "prompt_connect_failed_vehicle_locked" :  "Устройство находится в РЕЖИМЕ ПОТЕРИ. Пожалуйста, свяжитесь с нами для получения дополнительной информации." ,
+    "prompt_connect_failed_vehicle_lost_mode" :  "Устройство находится в РЕЖИМЕ ПОТЕРИ. Вы можете отключить его в разделе "Найти мое устройство", если вы являетесь его владельцем." ,
```
Comment (inmotion-global) [inline ru-RU.dart:169]: If you want to use double quotes in your translation copy, add the escape character backslash before the double quotes.

File: ru-RU.dart [hunk 5]
```diff
@@ -0,0 +1,845 @@
...
+    "title_vehicle_password" :  "Пароль" ,
+    "text_vehicle_password_disabled" :  "Отключено" ,
+    "text_vehicle_password_gesture" :  "Жест" ,
+    "text_vehicle_password_slide" :  "Слайд" ,
+    "text_vehicle_password_number" :  "Номер" ,
+    "title_modify_vehicle_password" :  "Изменить пароль" ,
+    "title_set_vehicle_password" :  "Установить пароль" ,
+    "prompt_vehicle_number_password_tips" :  "Пожалуйста, введите пароль из 4 или 6 цифр" ,
+    "title_vehicle_userinfo" :  "Информация о пользователе устройства" ,
+    "title_vehicle_userinfo_avatar" :  "Аватар"устройства" ,
```
Comment (inmotion-global) [inline ru-RU.dart:281]: Please remove extra double quotes from this line.

File: ru-RU.dart [hunk 6]
```diff
@@ -0,0 +1,845 @@
...
+    "vehicle_setting_riding_pedestrian_mode_tips" :  "Легче толкать". ,
+    "vehicle_setting_item_riding_mode_euc_normal" :  "Нормальный" ,
+    "vehicle_setting_item_riding_mode_euc_normal_tips" :  "Более безопасный, подходит для большинства пользователей." ,
+    "vehicle_setting_item_riding_mode_euc_fancier" :  "Скоростной" ,
+    "vehicle_setting_item_riding_mode_euc_fancier_tips" :  "Более высокая выходная мощность и более высокая максимальная скорость". ,
+    "prompt_cannt_set_sport_mode_when_limit_speed_25kmph_enabled" :  "Вы не можете переключиться в режим 'S', когда включено ограничение скорости на 25 км/ч." ,
+    "prompt_cannt_enable_limit_speed_25kmph_when_sport_mode_enabled" :  "Вы не можете включить «Ограничение скорости на 25 км/ч» в режиме «S». ,
+    "vehicle_setting_item_berm_angle_mode" :  "Режим угла" ,
+    "prompt_unlock_factory_safe_speed_limit" :  "Для вашей безопасности ограничение максимальной скорости может быть увеличено до 90 км/ч только тогда, когда ваш общий пробег превышает 30 км. %s" ,
+    «prompt_unlock_factory_safe_speed_limit_not_now» :
```
Comment (inmotion-global) [inline ru-RU.dart:366]: Please use double quotes instead of '《'.

File: ru-RU.dart [hunk 7]
```diff
@@ -0,0 +1,845 @@
...
... и решите принять или не принять это заявление. Вы не можете разблокировать ограничение максимальной скорости, пока не согласитесь с этим утверждением. Ваше \"Согласие\" будет считаться принятием данного отказа от ответственности. Если у вас есть какие-либо вопросы об этом отказе от ответственности, нажмите \"Отмена\"", Пожалуйста, не позволяйте никому, кто не прочитал инструкцию внимательно или не знаком с мерами предосторожности при езде на V13, управлять вашим устройством.\n5. Перед каждой поездкой обязательно проверяйте, находятся ли продукты и аксессуары в хорошем состоянии, избегайте использования неоригинальных аккумуляторов и аксессуаров, а также не разбирайте, не взламывайте и не модифицируйте продукты и аксессуары без разрешения. Вы несете ответственность за любые потери, ущерб и риски, вызванные неправильной операцией.\n\nПожалуйста, внимательно прочитайте и решите принять или не принять это заявление. Вы не можете разблокировать ограничение максимальной скорости, пока не согласитесь с этим утверждением. Ваше \"Согласие\" будет считаться принятием данного отказа от ответственности. Если у вас есть какие-либо вопросы об этом отказе от ответственности, нажмите \"Отмена\"",
```
Comment (inmotion-global) [inline ru-RU.dart:371]: Please complete the missing double quotes.

File: ru-RU.dart [hunk 8]
```diff
@@ -0,0 +1,845 @@
...
+    "prompt_motor_self_check_timeout" :  "Тайм-аут проверки" ,
+    "prompt_motor_self_check_failed" :  "Не удалось запустить проверку" ,
+
+    "title_vehicle_acc_dication" :  "Калибровка дроссельной заслонки" ,
+    "title_vehicle_brake_dication" :  "Калибровка тормозов" ,
+    "title_vehicle_gyro_dication" :  "Калибровка гироскопа" ,
+    "prompt_vehicle_acc_dication_tips" :  "После начала калибровки удерживайте ручку газа в максимальном и минимальном положении в течение 1 секунды. Убедитесь, что указанная операция калибровки завершена в течение 5 секунд." ,
+    "prompt_vehicle_brake_dication_tips" :  "После начала калибровки удерживайте ручку тормоза в максимальном и минимальном положении в течение 1 секунды. Убедитесь, что указанная операция калибровки завершена в течение 5 секунд." ,
+    "prompt_vehicle_gyro_dication_tips" :  "Пожалуйста, выпрямите самокат, чтобы подставка для ног оставалась параллельной земле, затем начните калибровку, пока она не завершится." ,
+    "prompt_cannt_start_vehicle_acc_dication_when_zero_start_enabled" :  "Вы не могли начать калибровку рукоятки газа, когда включен режим "Нулевой старт"!" ,
```
Comment (inmotion-global) [inline ru-RU.dart:407]: If you want to use double quotes in your translation copy, add the escape character backslash before the double quotes.

File: ru-RU.dart [hunk 9]
```diff
@@ -0,0 +1,845 @@
...
+    "text_firmware_update_notes_3" :  "· Если обновление не удалось, попробуйте выйти из приложения и выполнить обновление снова" ,
+    "prompt_contact_us" :  "Свяжитесь с нами" ,
+    "prompt_firmware_update_learn_more_notes" :  "Подробнее>>" ,
+    "title_new_firmware_version_summary" :  "Рекомендации" ,
+    "text_new_firmware_version_learn_more" :  "Подробнее>>" ,
+    "prompt_firmware_update_success" :  "Обновление произведено успешно" ,
+    "prompt_firmware_update_failed" :  "Ошибка обновления!" ,
+    "prompt_last_firmware_update_interrupted" :  "Последнее обновление прошивки могло быть прервано, пожалуйста, продолжайте обновление, чтобы исправить это." ,
+    "prompt_forced_firmware_update_tips" :  ">Вам необходимо обновить прошивку устройства до последней версии, прежде чем ее можно будет использовать" ,
+    "prompt_firmware_update_le1_manually_process_tips" :  "После отключения дисплея устройства подождите 3 секунды и включите его вручную. Затем нажмите "Подтвердить", чтобы продолжить обновление" ,
```
Comment (inmotion-global) [inline ru-RU.dart:496]: If you want to use double quotes in your translation copy, add the escape character backslash before the double quotes.

File: ru-RU.dart [hunk 10]
```diff
@@ -0,0 +1,845 @@
...
+    "bms_error_voltage_sample" :  "Ошибка выборки напряжения." ,
+    "bms_error_charge_current_sample" :  "Ошибка текущего образца заряда." ,
+    "bms_error_discharge_current_sample" :  "Ошибка сброса текущего образца." ,
+    "bms_error_temperature_sensor" :  "Ошибка датчика температуры." ,
+    "bms_error_rh_sensor" :  "Ошибка датчика относительной влажности." ,
+    "bms_error_charge_mosfet" :  "Ошибка зарядки mos." ,
+    "bms_error_discharge_mosfet" :  "Ошибка разряда mos." ,
+    "bms_error_afe" :  "Ошибка Afe." ,
+    "bms_error_cell" :  "Ошибка ячейки". ,
+    «bms_error_mcu» :  «Ошибка MCU». ,
```
Comment (inmotion-global) [inline ru-RU.dart:616]: Please use double quotes instead of '《'.

File: ru-RU.dart [hunk 11]
```diff
@@ -0,0 +1,845 @@
...
+    'prompt_light_effect_edit_already_first_step' :  "Это уже первый шаг!" ,
+    'prompt_light_effect_edit_already_last_step' :  "Это уже последний шаг!" ,
+    'prompt_light_effect_drag_to_delete' :  'Перетащите сюда, чтобы удалить' ,
+    'prompt_light_effect_release_to_delete' :  'Отпустите, чтобы удалить' ,
+    "title_vehicle_light_effect_exists" :  "Световой эффект в устройство" ,
+    "text_light_effect_default_package" :  "Световой эффект по умолчанию" ,
+    "text_light_effect_unknown_package" :  "Неизвестный световой эффект" ,
+    "prompt_enable_light_effect_failed" :  "Не удалось включить световой эффект" ,
+    'prompt_confirm_to_delete_light_effect' :  'Вы подтверждаете удаление?' ,
+    'prompt_confirm_to_delete_diy_light_effect' :  "Световой эффект "Сделай сам" не может быть восстановлен после удаления. Вы подтверждаете удаление "%s"?" ,
```
Comment (inmotion-global) [inline ru-RU.dart:682]: If you want to use double quotes in your translation copy, add the escape character backslash before the double quotes.

File: ru-RU.dart [hunk 12]
```diff
@@ -0,0 +1,845 @@
...
+    "title_complete_sound_pack_info" :  "Информация о звуковом пакете" ,
+    "btn_compose" :  "Компилировать" ,
+    "title_sound_pack_compose" :  "Компилировать звуковой пакет" ,
+    "prompt_sound_pack_compose_succeed" :  "Завершено!" ,
+    "prompt_sound_pack_compose_failed" :  "Не удалось создать" ,
+    "title_sound_pack_audition" :  "Прослушивание" ,
+    "title_select_sound_pack_item" :  "Выбрать звук" ,
+    'prompt_confirm_to_delete_sound_pack' :  'Вы подтверждаете удаление?' ,
+    'prompt_confirm_to_delete_sound_pack_in_vehicle' :  "Звуковой пакет в устройстве не может быть восстановлен после удаления. Вы подтверждаете удаление "%s"?" ,
+    'prompt_confirm_to_delete_diy_sound_pack' :  "Не удалось восстановить звуковой пакет "Сделай сам" после удаления. Вы подтверждаете удаление "%s"?" ,
```
Comment (inmotion-global) [inline ru-RU.dart:711]: If you want to use double quotes in your translation copy, add the escape character backslash before the double quotes.

File: ru-RU.dart [hunk 13]
```diff
@@ -0,0 +1,845 @@
...
+    'title_signature' :  'Подпись' ,
+    'title_modify_nickname' :  'Изменить никнейм' ,
+    'title_modify_signature' :  'Изменить подпись' ,
+    'prompt_nickname_limit' :  '6-20 символов' ,
+    'prompt_signature_limit' :  '0-100 символов' ,
+
+    /// my vehicle
+    'title_my_vehicles' :  'Мои устройства' ,
+    'title_delete_authorized_vehicle' :  'Удалить авторизацию владельца' ,
+    'prompt_delete_authorized_vehicle_tips' :  'Удаление устройства приведет к удалению авторизации его владельца. Вы подтверждаете?» ,
```
Comment (inmotion-global) [inline ru-RU.dart:767]: Please use double quotes instead of '》'.

**Review by inmotion-global — COMMENTED** [review]
Please fix the issues mentioned above and PR again, thank you very much.

**Comment by ValeraVvV** [comment]
complite

ср, 1 мар. 2023 г. в 15:04, inmotion-global ***@***.***>:

**Pull request closed without merging**